apps/server/src/modules/authentication/controllers/dto/ldap-authorization.params.ts

```diff
@@ -0,0 +1,22 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsMongoId, IsNotEmpty, IsString } from 'class-validator';
+
+export class LdapAuthorizationParams {
```

Comment:
In other places of the code we distinguish between `<name>BodyParams`, `<name>UrlParams` and `<name>QueryParams`. Maybe we should do that here as well? (The files names would then have to be renamed accordingly, e.g. `<name>.body.params.ts`.